client/webserver/webserver.go

```diff
@@ -625,6 +627,37 @@ func New(cfg *Config) (*WebServer, error) {
 	return s, nil
 }
 
+// fetchLatestVersion is a helper function to retrieve the latest version of the app
+// from github.
+func (w *WebServer) fetchLatestVersion(ctx context.Context) {
+	if w.latestVersion != "" {
+		return // already set
+	}
+
+	ticker := time.NewTicker(30 * time.Second)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			var response struct {
+				TagName string `json:"tag_name"`
+			}
+
+			err := dexnet.Get(ctx, "https://api.github.com/repos/decred/dcrdex/releases/latest", &response, dexnet.WithSizeLimit(1<<22))
+			if err != nil {
+				log.Errorf("Error getting latest version: %v", err)
+				continue
+			}
+
+			w.latestVersion = strings.Trim(response.TagName, "v")
```

Comment:
It's possible this isnt just v1.x.x https://api.github.com/repos/decred/dcrd/releases/latest

Reply:
Possible yes but we’ll have to work with how dcrdex names releases, not sure why that should change.

Reply:
Errm, prior to v0.6.x it was name differently "DCRDEX v0.x.x" but going forward I think we should stick to one way of doing things, using the vx.x.x as tag name.

Reply:
https://github.com/decred/dcrdex/pull/3286/commits/568af47199fde6124302a560c628b317594125c4 introduced gt check that should resolve your concerns @JoeGruffins.

Reply:
Should proly use a regex here to future-proof this.

Review comment: https://github.com/decred/dcrdex/pull/3286#discussion_r2106638527